package/MDAnalysis/analysis/base.py

```diff
@@ -103,16 +103,41 @@
    contains a step-by-step example for writing analysis tools with
    :class:`AnalysisBase`.
 
-
 .. _`Writing your own trajectory analysis`:
    https://userguide.mdanalysis.org/stable/examples/analysis/custom_trajectory_analysis.html
 
 
+If your analysis is operating independently on each frame, you might consider
+making it **parallelizable** via adding a :meth:`get_supported_backends` method,
+and appropriate aggregation function for each of its results. For example, if
+you have your :meth:`_single_frame` method storing important values under
+:attr:`self.results.timeseries`, you will write:
+
+.. code-block:: python
+
+    class MyAnalysis(AnalysisBase):
+        @classmethod
+        def get_supported_backends(cls):
+            return ('serial', 'multiprocessing', 'dask',)
+
+        _analysis_algorithm_is_parallelizable = True
```

Comment:
Whilst there's no guidance for this, I would strongly suggest putting class variables at the top of the class (something which I think we've been reasonably good at doing - see our various writers / readers: https://github.com/MDAnalysis/mdanalysis/blob/develop/package/MDAnalysis/coordinates/LAMMPS.py#L154).

Making sure we have keep this consistency in our documentation would be good.

```suggestion
        _analysis_algorithm_is_parallelizable = True

        @classmethod
        def get_supported_backends(cls):
            return ('serial', 'multiprocessing', 'dask',)

```